internal/controller/constants.go

```diff
@@ -45,6 +45,16 @@ const (
 	//RedisServerImageDefault = "quay.io/openshift/lightspeed-service-redis:latest"
 	// OLSConfigHashKey is the key of the hash value of the OLSConfig configmap
 	OLSConfigHashKey = "hash/olsconfig"
+	// OLSAppTLSHashKey is the key of the hash value of the OLS App TLS certificates
+	OLSAppTLSHashKey = "hash/olstls"
+	// OLSCAHashKey is the key of the hash value of the OLS CA certificate
+	OLSCAHashKey = "hash/ca"
+	// OLSConsoleTLSHashKey is the key of the hash value of the OLS Console TLS certificates
+	OLSConsoleTLSHashKey = "hash/olsconsoletls"
+	// RedisTLSHashKey is the key of the hash value of Redis
+	RedisTLSHashKey = "hash/redistls"
+	// RedisCAHashKey is the key of the hash value of Redis CA Cert
+	RedisCAHashKey = "hash/redisca"
```

Comment:
we can remove `RedisTLSHashKey` and `RedisCAHashKey` for they are no longer needed.

Reply:
The idea was to keep them as is and use them for our new cache. If we want them to be removed, lets take them in a separate PR as we have many other things to remove regarding redis in our current mainstream. Is that okay?

Reply:
ok we leave it there